src/services/bookService.js

```diff
@@ -0,0 +1,16 @@
+const bookDao = require("../models/bookDao");
+
+const getBooksById = async (id) => {
+  const books = await bookDao.getBooksById(id);
+  return books;
+}; 
+
+const getBooksByName = async (title) => {
+  const books = await bookDao.getBooksByName(title);
+  return books;
+};
```

Comment:
이전 pr과 통일성을 맞추기 위해 다음과 같이 수정해주세요!
```suggestion
const getBooksByName = async (title) => {
  return await bookDao.getBooksByName(title);
};
```

Reply:
수정 완료하였습니다!